### ISSUE TITLE
Update Contract Constructors 

### ISSUE THREAD

**Issue by joshma91** [truffle-box/react-auth-box#40]
Solidity has deprecated defining constructors with the same name as the contract. This warning applies to `Ownable.sol` and `Killable.sol`.
```
Warning: Defining constructors as functions with the same name as the contract is deprecated. Use "constructor(...) { ... }" instead.
  function Ownable() {
  ^ (Relevant source part starts here and spans across multiple lines).
,/home/josh/dev/react-auth-box/contracts/zeppelin/ownership/Ownable.sol:13:3: Warning: No visibility specified. Defaulting to "public". 
  function Ownable() {
  ^ (Relevant source part starts here and spans across multiple lines).
,/home/josh/dev/react-auth-box/contracts/zeppelin/ownership/Ownable.sol:22:3: Warning: No visibility specified. Defaulting to "public". 
  function transferOwnership(address newOwner) onlyOwner {
  ^ (Relevant source part starts here and spans across multiple lines).
,/home/josh/dev/react-auth-box/contracts/zeppelin/lifecycle/Killable.sol:12:3: Warning: No visibility specified. Defaulting to "public". 
  function kill() onlyOwner {
  ^ (Relevant source part starts here and spans across multiple lines).
```
